CircuitPy/FC_Board/lib/pysquared.py

```diff
@@ -272,6 +272,34 @@ def reset_vbus(self):
         except Exception as e:
             self.debug_print('vbus reset error: ' + ''.join(traceback.format_exception(e)))
 
+    @property
+    def gyro(self):
+        try:
+            return self.imu.gyro
+        except Exception as e:
+            self.debug_print('[ERROR][GYRO]' + ''.join(traceback.format_exception(e)))
+
+    @property
+    def imu_accel(self):
+        try:
+            return self.imu.acceleration
+        except Exception as e:
+            self.debug_print('[ERROR][ACCEL]' + ''.join(traceback.format_exception(e)))
+
+    @property
+    def temp(self):
```

Comment:
If you're planning to call the onboard temp sensor you should call it something like ```imu_temp``` to differentiate it from other sensors that might be on the board.